redis/reply_test.go

```diff
@@ -135,6 +151,61 @@ func TestReply(t *testing.T) {
 	}
 }
 
+func TestSlowLog(t *testing.T) {
+	c, err := dial()
+	if err != nil {
+		t.Errorf("TestSlowLog failed during dial with error " + err.Error())
+		return
+	}
+	defer c.Close()
+
+	result, err := c.Do("CONFIG", "GET", "SLOWLOG-LOG-SLOWER-THAN")
```

Comment:
This fails for me on redis 3.0.6 as the variable is lower case. This will require all the uses of this below to change too.

We should also protect against versions not supporting the command by skipping the test if this returns nothing.

Reply:
Agree. Fixed it.